core/src/main/java/org/hibernate/ogm/service/impl/OgmIntegrator.java

```diff
@@ -85,6 +114,17 @@ private void doIntegrate(Configuration configuration, SessionFactoryImplementor
 		attachPersistListener( serviceRegistry );
 	}
 
+	private boolean isOgmUsed(Map properties) {
+		// Integrator are discovered via the Java ServiceLoader mechanism and get executed independently of whether a
```

Comment:
Ah, very nice. I still meant to create a separate issue for it, but you are ahead of me :)